hooks/post_gen_project.py

```diff
@@ -42,6 +42,41 @@ def delete_gitlab_ci():
 def delete_jenkins_ci():
     pass
 
+def readme_toc_update():
+    with open("../{{ cookiecutter.project_short_name }}/README.md", "r") as fr:
+        with open("./temp.txt", "w") as tw:
+            tw.write('\n')
+            tw.write("<!TOC_START>\n\n")
+            lines = fr.readlines()
+            for line in lines:
+                if line.startswith("## ") and not line.startswith("## Table of Contents"):
```

Comment:
make sure that max line length is 89, use `black --line-length 79` to format it

Reply:
Done